kadi/commands/states.py

```diff
@@ -1822,12 +1834,12 @@ def fids_callback(msid, date, transitions, state, idx):
         else:
             # Only AFLCxxDy commands can make it here from set_transitions filtering
             fid_id = int(msid[4:6])
-            # state["fids"] could be None at the beginning
-            fids = state["fids"]
-            if fids is None:
-                fids = set()
+            # state["fids"] could be None at the beginning so wait until an AFLCRSET
+            # command to define it.
+            if state["fids"] is None:
+                return
```

Comment:
This change was not strictly necessary but makes this callback more consistent with others and makes the behavior more atomic.